examples/common/logs_foo_library/BUILD

```diff
@@ -0,0 +1,16 @@
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "common_logs_foo_library",
+    srcs = [
+        "foo_library.cc",
+    ],
+    hdrs = [
+        "foo_library.h",
+    ],
+    defines = ["BAZEL_BUILD"],
+    deps = [
+        "//api",
+        "//sdk:headers",
```

Comment:
why do we need sdk::headers, I see this included in earlier PRs too so probably I am missing something :)

Reply:
It's needed for `"opentelemetry/sdk/version/version.h"`.

Reply:
ok, thanks for letting me know. Not related to this PR, but we may have to revisit this afterward as ideally instrumentation libraries should have no dependency on SDK.